app/models/pipeline_run.rb

```diff
@@ -860,7 +859,31 @@ def update_job_status
       self.job_status = "#{prs.step_number}.#{prs.name}-#{prs.job_status}"
       self.job_status += "|#{STATUS_READY}" if report_ready?
     end
-    save
+    save!
+    enqueue_new_pipeline_run if automatic_restart
+  end
+
+  protected def report_failed_pipeline_run_stage(prs)
+    message = "SampleFailedEvent: Sample #{sample.id} by #{sample.user.admin? ? 'admin user' : 'non-admin user'} " \
```

Comment:
Do you want to incorporate this change here? :) 
https://github.com/chanzuckerberg/idseq-web/pull/2291/files
Then I'll close my PR

Reply:
Sure, it is a good idea. I can do that.

Reply:
Done